sources/install/package_network.sh

```diff
@@ -92,6 +92,17 @@ function install_nmap() {
     add-to-list "nmap,https://nmap.org,The Network Mapper - a powerful network discovery and security auditing tool"
 }
 
+function install_nmap-parse-output() {
+    colorecho "Installing nmap-parse-output"
```

Comment:
We'll remove the alias, meaning you need to add `# CODE-CHECK-WHITELIST=add-aliases` at the beginning of the function